ports/stm32/eth.c

```diff
@@ -258,22 +273,10 @@ static int eth_mac_init(eth_t *self) {
 
     #if defined(STM32H5)
     __HAL_RCC_ETH_RELEASE_RESET();
-
-    __HAL_RCC_ETH_CLK_SLEEP_ENABLE();
```

Comment:
I'll test again with 
    __HAL_RCC_ETH_CLK_SLEEP_DISABLE(); for these to be explicit, with comment why.